cli/archive_artifacts.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Strip dev packages
+yarn install --production --pure-lockfile
+
+tar cf build.tar \
```

Comment:
Can you please add a comment somewhere mentioning that we need `tar` in order to keep monorepo symlinks